admin/views/pupil-register/pupils-list-v2.ejs

```diff
@@ -0,0 +1,151 @@
+                        >
+                            <div class="govuk-highlight-wrapper">
+                                <% if (isFeatureEnabled('pupilEdit') && !availabilityData.hdfSubmitted && !availabilityData.checkWindowClosed) { %>
+                                    <a class="govuk-link govuk-!-font-weight-bold name-text-wrap" href="/pupil-register/pupil/edit/<%= p.urlSlug %>" ><%= p.fullName %></a>
+                                <% } else { %>
+                                    <span class="govuk-!-font-weight-bold name-text-wrap"><%= p.fullName %></span>
+                                <% } %>
+                                <% if (p.showDoB) { %>
+                                    <div class="govuk-font-greyed-out govuk-!-font-size-16">Date of birth: <%= p.dateOfBirth %></div>
+                                <% } %>
+                            </div>
+                        </td>
+                        <td class="govuk-table__cell"><%= p.group %></td>
+                    </tr>
+                <% }) %>
+                </tbody>
+            </table>
+            <div class="govuk-back-to-top">
+                <a class="govuk-link govuk-link--no-visited-state app-back-to-top__link" href="#top">
+                    <svg role="presentation" focusable="false" class="app-back-to-top__icon" xmlns="http://www.w3.org/2000/svg" width="13" height="17" viewBox="0 0 13 17">
+                        <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
+                    </svg>Back to top
+                </a>
+            </div>
+        </div>
+        <div class="govuk-grid-column-one-third" aria-label="sidebar">
+            <aside class="app-related-items" role="complementary">
+                <h2 class="govuk-heading-m" id="subsection-title">
+                    Related
+                </h2>
+                <nav role="navigation" aria-labelledby="subsection-title">
+                    <ul class="govuk-list govuk-!-font-size-16">
+                        <li>
+                            <a class="govuk-link" href="<%- guidancePdf %>" target="_blank" rel="noopener noreferrer">Guidance</a>
+                        </li>
+                        <li>
+                            <a class="govuk-link" href="/group/pupils-list" rel="noopener noreferrer">Group pupils</a>
+                        </li>
+                        <li>
+                            <a class="govuk-link" href="/pupils-not-taking-the-check/pupils-list" rel="noopener noreferrer">Pupils not taking the check</a>
+                        </li>
+                        <li>
+                            <a class="govuk-link" href="/access-arrangements/overview" rel="noopener noreferrer">Access arrangements</a>
+                        </li>
+                    </ul>
+                </nav>
+            </aside>
+        </div>
+    </div>
+</main>
+<script>
```

Comment:
We should be removing all inline script tags for security reasons.

Reply:
@jon-shipley this should be a separate piece of work that tackles the removal of all the script tags we currently have in our views.
Function calls are related to sorting, autocompletion, check forms, printing and access arrangements.

Reply:
but if you are just creating this page it would be best to do it here, rather than creating extra debt?